pkgs/retry/retry

```diff
@@ -4,6 +4,8 @@
 #
 set -euf
 
+export PATH=@coreutils@/bin"${PATH+:$PATH}"
```

Comment:
I don't know its name, but it's used by [substituteAll](https://github.com/zalora/microgram/pull/16/files#diff-9d168b32227c60d3951d0fd8cf125d22R11).